core/base/Module.php

```diff
@@ -188,21 +194,32 @@ public function init()
                 throw new InvalidConfigException(sprintf('The required component "%s" is not registered in the configuration file', $component));
             }
         }
-        
+    
+        $this->theme = Yii::$app->themeManager->getActiveTheme();
```

Comment:
As the theme config is hold, configured and setup in the theme component (Yii::$app->themeManager) why do we need another public $theme property? It might be confusing to have different places which hold the same object as now $this->theme would be equals to:
YIi::$app->themeManager->activeTheme.

Maybe introduce an `hasActiveTheme` method which ensures the active theme and then returns the them object.

```php
if (Yii::$app->themeManager->hasActiveTheme) {
    $this->layout = Yii::$app->themeManager->activeTheme->layout;
}
```

Reply:
Good point, thanks.